gen_linkspath/gen_linkspath.ml

```diff
@@ -0,0 +1,29 @@
+module C = Configurator.V1
+
+let check_opam : bool = 
+  try Sys.getenv "LINKS_BUILT_BY_OPAM" = "1" with _ -> false
```

Comment:
A minor nitpick: I'd prefer explicitly checking whether "LINKS_BUILT_BY_OPAM" is present in the environment, e.g. using Sys.getenv_opt, over anticipating that the missingness of the environment variable will trigger some unnamed exception.  Giving a name to the exception would be slightly better but I suggest just using Sys.getenv_opt.